addons/editor-devtools/userscript.css

```diff
@@ -440,7 +440,7 @@ div.s3devDDOut.vis ul.s3devDD {
   border-radius: 4px;
   box-shadow: rgba(0, 0, 0, 0.3) 0 0 3px, rgba(0, 0, 0, 0.2) 0 3px 10px;
 
-  z-index: 1000001;
+  z-index: 999;
```

Comment:
Can you explain this change? z-index is madness and even tho 999 might appear to work just fine at first, it might not in some edge case.

Reply:
It's so that dragged blocks appear above the floating find bar, which I think is better when the bar is not closed the instant a block is added.